packages/stream_chat/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.5.1
+
+🐞 Fixed
+- Unread count specific to channel setting global unread count on a specific case.
```

Comment:
This sentence is a bit unclear. Not too sure what this fixes

Reply:
```suggestion
- `channel.unreadCount` was being set as using global unread count on a very specific case.
```